### PR TITLE
Translate README to English

### PR DESCRIPTION
**Pull request by xzqvac**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,71 +1,69 @@
...
+The first step in our action plan will be to analyze the existing solution in practice. This aims to thoroughly examine the technical aspects of the OBD interface operation, such as the type of communication protocol, connector type, and so on.
+
+The next stage of the plan will be prototyping and implementing our own solution. This will primarily involve successfully reading information from the OBD system, such as speed, error codes, fuel status, etc. The next problem to be solved will be the issue of wireless communication, understanding and implementing Bluetooth and LoRa transmission protocols.
+
+Finally, the remaining time will be dedicated to refining the whole solution, ensuring its reliability, as well as optimizing the use of resources.
+
+# Summary
 
-Pobieranie danych z systemu OBD
+Retrieving data from OBD system using already done libraries:
```
Comment (yksen) [inline README.md:56]: with ready-to-use open-source libraries

File: README.md [hunk 2]
```diff
@@ -1,71 +1,69 @@
...
 - [Python](https://github.com/brendan-w/python-OBD)
 - [C++](https://github.com/stanleyhuangyc/ArduinoOBD)
 
-Udostępnianie pełnych danych diagnostycznych i statystycznych przy użyciu standardu Bluetooth.
+Sharing full diagnostic and statistical data using Bluetooth standard.
 
-Końcowym urządzeniem odbiorczym będzie telefon z system operacyjnym Android, na który zostanie napisana dedykowana aplikacja.
+The final receiving device will be an Android-based phone, on which a dedicated application will be written.
```
Comment (yksen) [inline README.md:62]: application will be developed

Comment (nebcio) [reply README.md:62]: . up

File: README.md (hunk 1 above)
Comment (nebcio) [reply README.md:56]: . up

**Review by yksen — APPROVED** [review]

**Pull request merged**